app/src/main/java/me/angrybyte/contactsgenerator/api/ContactOperations.java

```diff
@@ -241,12 +247,36 @@ public boolean deleteNextContact() {
         return true;
     }
 
+    private String generatePhoneNumber() {
+        Random rndGenerator = new Random();
```

Comment:
There's already a `rndPhoneTypeGenerator` which can be a generic random generator. There's no security concern with this app, I believe we can just reuse it 🙂